### ISSUE TITLE
how to train ssd with larger input resolution

### ISSUE THREAD

**Issue by chenjunweii** [tensorflow/models#6357]
I would like to train ssd mobilenet 512, I have modified the config like the following code, but the performance seems not much different compared to default  ssd mobilenet 300, is there any other places that i have to modify ? thanks

```
image_resizer {
       fixed_shape_resizer {
        height: 512
        width: 512
      }
}

```

**Comment by karolmajek** [comment]
I just yesterday changed 300x300 to 600x600 and I haven't noticed a performance drop in inference.
In the training, I changed the batch size to smaller one to still fit 4 GPUs

**Comment by rootkitchao** [comment]
If you want to try to get higher accuracy, it would be better to try Retinanet.The SSD model is limited by itself and has poor recognition ability for small objects.In the case of using Mobilenet V1 as the backbone, the ability to recognize large objects is not particularly good.Using Retinanet or RFBNet can greatly improve accuracy.Upgrading the backbone to Mnasnet can improve the ability to recognize large objects.
For your problem, modifying the fixed_shape_resizer has indeed changed the input resolution.Theoretical accuracy will increase slightly, but may be limited by the learning rate setting.

**Comment by chenjunweii** [comment]
@rootkitchao Is that so, I will try other networks as the backbone, thanks !

**Comment by ymodak** [comment]
Closing this issue since its resolved. Feel free to reopen if still have any further questions. Thanks!

**Comment by JayjieL** [comment]
i do change the image_resizer. but the result is non object will be detected.....
image_resizer {
       fixed_shape_resizer {
        height: 720
        width: 1280
      }
}

**Comment by SLAMgreen** [comment]
@chenjunweii Do you knewn how to change the backbone in tensorflow models?How to modify the config file?
